spikeinterface_gui/mainwindow.py

```diff
@@ -19,6 +19,7 @@ def __init__(self,
         label_definitions=None,
         displayed_unit_properties=None,
         extra_unit_properties=None,
+        mode="full"
```

Comment:
For the mode, I think I would prefer to make dict  with set of view to load or not.

But first I wanted to refactor a view to load ext on demand before introducing this.